### PR TITLE
SS-1950 Section level records for FAQs

### PR DESCRIPTION
**Pull request by KaustubhKumar05**

### CODE REVIEW

File: lib/algolia/algoliaUtils.js [hunk 1]
```diff
@@ -10,6 +10,16 @@ const {
     getCleanedContent
 } = require('./helper');
 
+const slugify = (text) =>
```
Comment (hardik1003) [inline lib/algolia/algoliaUtils.js:13]: should use lib instead

Comment (hardik1003) [reply lib/algolia/algoliaUtils.js:13]: encodeURI google karlo

File: lib/algolia/getRecords.js [hunk 2]
```diff
@@ -2,17 +2,19 @@ const fs = require('fs');
...
-    createRecords
+    createRecords,
+    getSectionContent
 } = require('./algoliaUtils');
 
 async function updateIndex() {
-    const appId = process.env.NEXT_PUBLIC_ALGOLIA_APP_ID
-    const adminKey = process.env.NEXT_PUBLIC_ALGOLIA_ADMIN_API_KEY
-    const algoliaIndex = process.env.NEXT_PUBLIC_ALGOLIA_INDEX
+    const appId = process.env.NEXT_PUBLIC_ALGOLIA_APP_ID;
```
Comment (hardik1003) [inline lib/algolia/getRecords.js:15]: did we create config change ticket for this?

File: lib/algolia/algoliaUtils.js [hunk 3]
```diff
@@ -21,26 +31,66 @@ exports.cacheContentAlias = function cacheContentAlias(propPath) {
...
-        keywords: getKeywords(fileContent),
-        headings: getHeadings(fileContent),
-        content: getCleanedContent(fileContent, contentAlias),
-        platformName: platform,
-        objectID: link
-    };
-    return fileRecord;
+    const headers = getHeadings(fileContent);
+
+    if (splitSectionLevel) {
```
Comment (hardik1003) [inline lib/algolia/algoliaUtils.js:67]:
```suggestion
  const result = splitSectionLevel ? getHeadings(fileContent) : fileContent
result.forEach(()=>{
const fileRecord = {...}
fileRecords.push(fileRecord)

})
return fileRecords
```

**Review by hardik1003 — CHANGES_REQUESTED** [review]

File: lib/algolia/getRecords.js (hunk 2 above)
Comment (KaustubhKumar05) [reply lib/algolia/getRecords.js:15]: No, it was added to Vercel directly

File: lib/algolia/algoliaUtils.js (hunk 3 above)
Comment (hardik1003) [reply lib/algolia/algoliaUtils.js:67]: check if the code duplication is avoidable

Comment (KaustubhKumar05) [reply lib/algolia/algoliaUtils.js:67]: Will refactor it

File: lib/algolia/getRecords.js (hunk 2 above)
Comment (KaustubhKumar05) [reply lib/algolia/getRecords.js:15]: Created a ticket

**Review by bmbshlly — APPROVED** [review]

**Pull request merged**